src/pages/Footer.tsx

```diff
@@ -0,0 +1,192 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { makeStyles, Box, Button, Container } from "@material-ui/core";
+import FacebookIcon from "@material-ui/icons/Facebook";
+import TwitterIcon from "@material-ui/icons/Twitter";
+import InstagramIcon from "@material-ui/icons/Instagram";
+
+const useStyles = makeStyles(theme => ({
+  boxMd: {
+    display: "flex",
+    height: "20vh",
+    justifyContent: "center",
+    padding: "40px 80px",
+    backgroundColor: "#243867",
+    color: "white",
+    fontSize: "10px",
+    [theme.breakpoints.only("xs")]: {
+      display: "none"
+    }
+  },
+  boxSm: {
+    display: "none",
+    justifyContent: "center",
+    padding: "40px 80px",
+    backgroundColor: "#243867",
+    color: "white",
+    fontSize: "10px",
+    [theme.breakpoints.only("xs")]: {
+      display: "flex"
+    }
+  },
+  column: {
+    display: "flex",
+    justifyContent: "space-around",
+    alignItems: "center",
+    flexDirection: "column",
+    height: "20vh",
+    width: "100vw"
+  },
+  link: {
+    textDecoration: "none",
+    color: "white"
+  },
+  button: {
+    fontWeight: 400,
+    backgroundColor: "#5a698e",
+    color: "white",
+    padding: "5px 15px"
+  },
+  header: {
+    fontSize: "13px",
+    marginBottom: "10px"
+  },
+  logo: {
+    fontSize: "20px",
+    fontWeight: 500,
+    marginBottom: "5px"
+  },
+  icons: {
+    marginBottom: "10px"
+  },
+  icon: {
+    marginLeft: "5px"
+  },
+  container: {
+    padding: "5px"
+  },
+  container_first: {
+    display: "flex",
+    flexDirection: "column"
+  },
+  container_last: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center"
+  }
+}));
+
+const Footer = () => {
+  const styles = useStyles();
+
+  return (
+    <>
+      <Box className={styles.boxMd}>
+        <Container className={styles.container_first}>
+          <Box className={styles.logo}>MedLine.io</Box>
+          <Box className={styles.icons}>
+            <FacebookIcon className={styles.icon} />
+            <TwitterIcon className={styles.icon} />
+            <InstagramIcon className={styles.icon} />
+          </Box>
+          <Box>© MedLine.io 2020</Box>
+        </Container>
+        <Container className={styles.container}>
+          <Box className={styles.header}>FIGHT COVID</Box>
+          <Box>Nedds: Medical Supplies</Box>
+          <Box>Leads: Suppliers</Box>
+          <Box>
+            <Link className={styles.link} to="/login">
+              Log In
+            </Link>
+          </Box>
```

Comment:
think you can declare this as a mini component before the return() to remove code duplication :), like:

const BoxWithLink = (title, path) => (
<Box>
            <Link className={styles.link} to="/login">
              Log In
            </Link>
          </Box>
)

Reply:
Done